src/main/java/sda/games/chess/Player.java

```diff
@@ -0,0 +1,59 @@
+package sda.games.chess;
+
+import java.util.Scanner;
+
+public class Player {
+
+    public void getMove() {
+
+        String inputMove = "";
+
+        Scanner scanner = new Scanner(System.in);
+        System.out.println("Wprowadz ruch wpisujac pole z ktorego chcesz przeunac pionek na pole na ktore chcesz "
+                            + "na pole na ktorre chcesz przesunac wstawiajac pomiedzy swoj wybor znak \">\", np:\n"
+                            + "A1>H8");
+        System.out.print("Ruch: " + inputMove);
+        inputMove = scanner.nextLine();
+        int fromX = Integer.parseInt(charToInt(inputMove.toUpperCase().charAt(0)));
+        int fromY = Integer.parseInt(inputMove.substring(1, 2));
+        int toX  = Integer.parseInt(charToInt(inputMove.toUpperCase().charAt(4)));
+        int toY = Integer.parseInt(inputMove.substring(5));
+        
+        Move move = new Move(fromX, fromY, toX, toY);
+
+    }
```

Comment:
górna cześć super, dolną zrobiłabym nie za pomocą switch case

Reply:
public int charToInt (char c ) {
return (int) c - 96; // czy tam 97 nie wiem ile to jest żeby wyszło 

}

Reply:
👍 👍